torch/ao/sparsity/_variables.py

```diff
@@ -0,0 +1,25 @@
+import torch
```

Comment:
Suggest calling this sparsity_mappings.py, similar to: 
https://github.com/pytorch/pytorch/blob/master/torch/quantization/quantization_mappings.py